src/portal/src/app/project/list-project/list-project.component.ts

```diff
@@ -208,6 +211,10 @@ export class ListProjectComponent implements OnDestroy {
                     this.clrLoad(st);
                     this.statisticHandler.refresh();
                 }
+            }, error => {
+                this.translate.get("BATCH.DELETED_FAILURE").subscribe(res => {
```

Comment:
If you must have it here, it is better to use the back-end error message directly, just like the first code

Reply:
If delteting 3 projects,  2 fail, error message will be a Array.   MessageHandler can not handle this.